### PR TITLE
Added descriptions

### PR DESCRIPTION
**Pull request by manjilgtm**

### CODE REVIEW

**Review by tsalako — CHANGES_REQUESTED** [review]
Looks good. Can you include some screenshots?

**Comment by manjilgtm** [comment]
bottler

File: src/main/webapp/about.jsp [hunk 1]
```diff
@@ -27,33 +27,37 @@
...
-              side of things, but if you're particularly interested you might use
-              HTML, CSS, and JavaScript to make the chat app prettier.</li>
-            <li><strong>Customization:</strong> Think about a group you care about.
-              What needs do they have? How could you help? Think about technical
-              requirements, privacy concerns, and accessibility and
-              internationalization.</li>
+            <li><strong>Changing the User Description: </strong> To change description: change description to" NewDescription". Along with updating description, it
+            also gives link to profile page.</li>
+            <li><strong>Creating new conversation: </strong>To create new conversation:  create conversation "newConversationName"</li>
+            <li><strong>Send message:</strong> To send message to particular conversation: send message to "conversationTitle</li>
```
Comment (tsalako) [inline src/main/webapp/about.jsp]: missing a quotation mark here.

File: src/main/webapp/about.jsp [hunk 2]
```diff
@@ -27,33 +27,37 @@
...
-                and also make changes for performance and scalability as your app
-                increases in complexity.</li>
-            <li><strong>Look and feel:</strong> The focus of CodeU is on the Java
-              side of things, but if you're particularly interested you might use
-              HTML, CSS, and JavaScript to make the chat app prettier.</li>
-            <li><strong>Customization:</strong> Think about a group you care about.
-              What needs do they have? How could you help? Think about technical
-              requirements, privacy concerns, and accessibility and
-              internationalization.</li>
+            <li><strong>Changing the User Description: </strong> To change description: change description to" NewDescription". Along with updating description, it
```
Comment (tsalako) [inline src/main/webapp/about.jsp:35]: Missing descriptions:

NAVIGATE_TO_CONVERSATION 
GET_STAT
HELP
GET_CONVERSATIONS_ABOUT_KEYWORD

I may have missed some myself so just double check I guess.

File: src/main/webapp/about.jsp [hunk 3]
```diff
@@ -27,33 +27,37 @@
...
+            also gives link to profile page.</li>
+            <li><strong>Creating new conversation: </strong>To create new conversation:  create conversation "newConversationName"</li>
+            <li><strong>Send message:</strong> To send message to particular conversation: send message to "conversationTitle</li>
+            <li><strong>Retrieve message:</strong> To retrieve messages based on Conversation title: get messages from "conversationTitle"</li>
+            <li><strong>Navigate to any page:</strong> To navigate to any page: navigate to pageName</li>
```
Comment (tsalako) [inline src/main/webapp/about.jsp]: can you give example page names like "about page" or "activity page"

File: src/main/webapp/about.jsp [hunk 4]
```diff
@@ -27,33 +27,37 @@
...
+            <li><strong>Get all messages: </strong>To get all message user have sent: get my messages</li>
+            <li><strong>Retrieve conversation based on Time: </strong>To get conversation based on time: get conversation
+              made 12:00 AM (or any other Time expressions)</li>
+            <li><strong>Retrieve conversation: </strong> To retrieve conversation based on Author name: get conversation by "AuthorName"</li>
+            <li><strong>Creating new conversation: </strong> Create conversation "newConversationName"</li>
+            <li><strong>Retrieve all conversation: </strong> To get all conversation: Get all conversation</li>
+            <li><strong>Search Keywords: </strong> To search keyword in message: get message about "keyName"</li>
```
Comment (tsalako) [inline src/main/webapp/about.jsp]: **keyword** instead of **keyName**

File: src/main/webapp/about.jsp [hunk 5]
```diff
@@ -27,33 +27,37 @@
...
+            <li><strong>Creating new conversation: </strong> Create conversation "newConversationName"</li>
```
Comment (tsalako) [inline src/main/webapp/about.jsp]: this is in here twice.

**Review by tsalako — CHANGES_REQUESTED** [review]

File: src/main/webapp/about.jsp (hunk 1 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp]: Done!

File: src/main/webapp/about.jsp (hunk 4 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp]: Done!

File: src/main/webapp/about.jsp (hunk 5 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp]: Removed!

File: src/main/webapp/about.jsp (hunk 2 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp:35]: Updated!

File: src/main/webapp/about.jsp (hunk 3 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp]: added!

File: src/main/webapp/about.jsp [hunk 6]
```diff
@@ -35,17 +35,19 @@
...
             <li><strong>Retrieve conversation based on Time: </strong>To get conversation based on time: get conversation
               made 12:00 AM (or any other Time expressions)</li>
             <li><strong>Retrieve conversation: </strong> To retrieve conversation based on Author name: get conversation by "AuthorName"</li>
             <li><strong>Creating new conversation: </strong> Create conversation "newConversationName"</li>
             <li><strong>Retrieve all conversation: </strong> To get all conversation: Get all conversation</li>
-            <li><strong>Search Keywords: </strong> To search keyword in message: get message about "keyName"</li>
-            <li><strong>Creating new conversation: </strong> Create conversation "newConversationName"</li>
+            <li><strong>Search Keywords: </strong> To search keyword in message: get message about "keyword"</li>
+            <li><strong>For Help: </strong> To get information about Bot functions:Get Help</li>
+            <li><strong>Get conversation: </strong> To get COnversation that contains specific keyword: get conversation about "keyword"</li>
```
Comment (tsalako) [inline src/main/webapp/about.jsp]: COnversation to Conversation

**Review by tsalako — APPROVED** [review]

File: src/main/webapp/about.jsp (hunk 6 above)
Comment (manjilgtm) [reply src/main/webapp/about.jsp]: done!

**Pull request merged**